eventbrite.py

```diff
@@ -0,0 +1,430 @@
+        "auto_hide_before": "",
+        "auto_hide_after": "",
+        "sales_channels": ["online","atd"],
+        "hide_sale_dates": "false",
+        "delivery_methods": ["electronic"],
+        "include_fee": "false",
+        "sales_start": "2023-11-09T08:00:00Z",
+        "sales_end": "2023-11-11T03:00:00Z",
+        }
+    }
+
+def get_organization(http):
+    url = f"{EVENTBRITE_API_BASE_URL}users/me/organizations/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status != 200:
+        print(f"HTTP Error: {resp.status}")
+        
+    data = resp.data.decode('utf-8')
+    print(data)
+    return data
+
+def get_event_by_id(http, event_id):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status != 200:
+        print(f"HTTP Error: {resp.status}")
+        
+    data = resp.data.decode('utf-8')
+    print(data)
+    return data
+
+def get_events_by_organization(http, organization_id):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events"
+    headers = {
+        'Authorization': eventbrite_api_token
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status != 200:
+        print(f"HTTP Error: {resp.status}")
+    data = resp.data.decode('utf-8')
+    print(data)
+        
+def create_event(http, organization_id ,event_details):
+    url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    'Content-Type': 'application/json'
+    }
+
+    try:
+        # Make the POST request
+        resp = http.request('POST', url, headers=headers, body=event_details)
+    except urllib3.exceptions.HTTPError as e:
+        pass
+
+    # Check the status code to ensure a successful request
+    if resp.status != 200:
+        error_data = json.loads(resp.data.decode('utf-8'))
+        error_detail = error_data.get('error_detail', {})
+        return {
+            "error": error_data.get('error', 'Unknown Error'),
+            "error_description": error_data.get('error_description', 'An error occurred'),
+            "error_detail": error_detail,
+            "status_code": resp.status
+        }
+    data = resp.data.decode('utf-8')
+    print(data)
+    return json.loads(data)
+
+
+def update_event(http, event_id, event_details):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    'Content-Type': 'application/json'
+    }
+
+    try:
```

Comment:
actually why even catch the error? provided the request sent is correct,  we should expect no errors. If for some reason the usual request don't work anymore, I want this exception to occur and not die silently. Remove the try/except.